src/Bootstrap/Tab.elm

```diff
@@ -471,32 +471,41 @@ renderLink_ :
     -> (String -> State -> msg)
     -> Html.Html msg
 renderLink_ id active (Link { attributes, children }) configRec strToMsg =
+    let
+        commonClasses =
+            [ ( "nav-link", True ), ( "active", active ) ]
+
+        clickHandler =
+            onClick <|
```

Comment:
Good change, but I'm curious: Was using `onClick` just an improvement or was the custom part not working the or part of the issue? Might answer this myself as I read on, but feel free to answer!

Reply:
The custom/json.succeed part was definitely causing the issue.